deploy/llm/templates/scripts.yaml

```diff
@@ -7,6 +7,13 @@ metadata:
 data:
   start.sh: |
     #!/bin/sh
+    echo "model=${MODEL_NAME}"
+    country=`curl https://ifconfig.io/country_code`
```

Comment:
Sometimes it will give you a wrong answer, for e.g, a CN ip is treated as a foreign ip, when this happens, it still timeouts when visiting huggingface, you can have a timeout policy and switch it to modescope when huggingface timeouts.